curriculum/challenges/english/25-front-end-development/lab-recipe-page/668f08ea07b99b1f4a91acab.md

```diff
@@ -89,7 +89,7 @@ assert.lengthOf(document.querySelectorAll('h1'), 1)
 You should have a `p` element below your `h1` element with text describing your recipe.
 
 ```js
-assert($('p')[0].previousElementSibling.tagName === 'H1');
+assert(document.querySelector('p')?.previousElementSibling.tagName === 'H1');
```

Comment:
```suggestion
assert.strictEqual(document.querySelector('p')?.previousElementSibling.tagName, 'H1');
```

This would be an example of a specific assertion method. Please update the others as well.

Reply:
on it!